core/src/main/resources/defaults/clustertemplates/7.2.15/aws/search-analytics.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "7.2.15 - Search Analytics for AWS",
+  "description": "",
+  "type": "DISCOVERY_DATA_AND_EXPLORATION",
+  "featureState": "PREVIEW",
+  "cloudPlatform": "AWS",
+  "distroXTemplate": {
+    "cluster": {
+      "blueprintName": "7.2.15 - Search Analytics"
+    },
+    "instanceGroups": [
+      {
+        "name": "manager",
+        "nodeCount": 1,
+        "recoveryMode": "MANUAL",
+        "template": {
+          "attachedVolumes": [
+            {
+              "count": 1,
+              "size": 100,
+              "type": "standard"
```

Comment:
This is magnetic storage that will host the CM DB (embedded PostgreSQL since the `externalDatabase` bits are missing; see [7.2.14 Data Eng HA AWS example](https://github.com/hortonworks/cloudbreak/blob/8bebc0dee0b2662e559f4c0716a5f023a8f467a0/core/src/main/resources/defaults/clustertemplates/7.2.14/aws/dataengineering-ha.json#L11-L13) ). It would also host HDFS, but that is out of scope here. Are we sure this will not cause any performance issues with CM?

Reply:
This template is based on the DDE template and I took this setting over without modification. Additionally, most other templates seem to use the standard type for CM as well.